lib/cartopy/io/img_tiles.py

```diff
@@ -131,14 +134,12 @@ def tile_bbox(self, x, y, z, y0_at_north_pole=True):
         Returns the ``(x0, x1), (y0, y1)`` bounding box for the given x, y, z
         tile position.
 
-        Parameters
-        ----------
-        x, y, z : int
+        :param x, y, z : int
```

Comment:
Aren't you using Google-style docstrings all over?

Reply:
Yeah, missed this one.